module/actor.js

```diff
@@ -177,6 +177,9 @@ class DCCActor extends Actor {
     const init = this.data.data.attributes.init.value
     const roll = new Roll('@die+@init', { die, init })
 
+    //evaluate roll, otherwise roll.total is undefined
+    roll.evaluate()
```

Comment:
Looks good, but this needs a slight tweak to avoid a deprecation warning on newer versions.

I'll take a proper look later. Thanks!